merkletree/policy.go

```diff
@@ -8,6 +8,12 @@ import (
 
 type TimeStamp uint64
 
+// Policies is a summary of the directory's
+// current security policies. This includes the public part
+// of the VRF key used to generate private indices,
+// the cryptographic algorithms in use, as well as
+// the protocol version number.
+// See section 3.3 per paper for an detail example.
```

Comment:
s/per paper for an detail/of the CONIKS paper for a detailed

I don't want to assume that a developer reading this necessarily knows which paper we're referring to.